recipes/zfp/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+import os
+from conans import ConanFile, tools, CMake
+
+
+class ZfpConan(ConanFile):
+    name = "zfp"
+    description = "Compressed numerical arrays that support high-speed random access"
+    homepage = "https://github.com/LLNL/zfp"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-3-Clause"
+    topics = ("conan", "zfp", "compression", "arrays")
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = ["CMakeLists.txt"]
+    generators = ["cmake", "cmake_find_package"]
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_cuda": [True, False],
+        "with_bit_stream_strided": [True, False],
+        "with_aligned_alloc": [True, False],
+        "with_cache_twoway": [True, False],
+        "with_cache_fast_hash": [True, False],
+        "with_cache_profile": [True, False],
+        "with_openmp": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_cuda": False,
+        "with_bit_stream_strided": False,
+        "with_aligned_alloc": False,
+        "with_cache_twoway": False,
+        "with_cache_fast_hash": False,
+        "with_cache_profile": False,
+        "with_openmp": False,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
```

Comment:
```suggestion
            del self.options.fPIC
        if self.options.with_cuda:
            self.output.warn("Conan package for CUDA is not available,'
                             ' this package will be used from system.'")
        if self.options.with_openmp:
            self.output.warn("Conan package for OpenMP is not available,'
                             ' this package will be used from system.'")

```

I suggest you add an explicit warning about CUDA and OpenMP. There is an effort over llvm-openmp only for packaging.